docs/messaging/limitations.md

```diff
@@ -0,0 +1,15 @@
+---
+id: limitations
+title: Limitations
+slug: /messaging/limitations
+---
+
+### History state of past experiments ###
+
+It is currently not possible to enroll users into experiments based on previous experiments that they might have been a part of. If multiple messaging campaigns go out as experiments targeting different surfaces it is not possible to ensure that users will not see both campaigns.
+Mitigation is possible through the use of groups for frequency capping. If the experiments happen simultaneously then using the same group can ensure we control the time it takes for a user to see both campaigns.
+
+Similarly (using targeting) it is possible to prevent a user from seeing a message if impressions from a different message are present. However once a message has been removed (or the experiment has ended) message impressions are removed. This is only possible if the two messages overlap in the system at the same time.
```

Comment:
This is a good capture of known limitations. I believe message impressions live for experiments when part of message group , @Mardak can confirm so may be we can call out something like  'However once a message (not part of message group) has been removed (or the experiment has ended) message impressions are removed.'